snc.sh

```diff
@@ -169,6 +169,10 @@ fi
 # Wait for install to complete, this provide another 30 mins to make resources (apis) stable
 ${OPENSHIFT_INSTALL} --dir ${INSTALL_DIR} wait-for install-complete ${OPENSHIFT_INSTALL_EXTRA_ARGS}
 
+# Remove the openshift-authenticator CSR to force a certificate regeneration
+# BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1978193
+retry ${OC} delete csr system:openshift:openshift-authenticator
+
```

Comment:
we made a 4.8 release without this, I'm not sure what is the impact of this? Surely this deserves to be in a PR of its own?

Reply:
@cfergeau we made release of 4.8 using this https://github.com/code-ready/snc/commit/c1009676b800401cb572062715459da944ddb3d4 but for 4.9 bug is recently resolved so we don't need this change anymore.